### ISSUE TITLE
Stack trace in content-service on unrecognized content ID

### ISSUE THREAD

**Issue by smashwilson** [deconst/deconst-docs#59]
When the Cloud Files download returns a 404, it looks like pkgcloud is giving us the actual 404 HTML page as a response body, which we're attempting to parse as JSON:

```
debug: Requesting content ID: [https://github.com/smashwilson/deconst-drc-docs/blog]
debug: Downloading [https%3A%2F%2Fgithub.com%2Fsmashwilson%2Fdeconst-drc-docs%2Fblog] from container [deconst-drc-content]
debug: Final content: <html><h1>Not Found</h1><p>The resource could not be found.</p></html>
error: SyntaxError: Unexpected token <
     at Object.parse (native)
     at Request.<anonymous> (/usr/src/app/src/content.js:34:23)
     at Request.emit (events.js:129:20)
    at IncomingMessage.<anonymous> (/usr/src/app/node_modules/pkgcloud/node_modules/request/request.js:998:12)
    at IncomingMessage.emit (events.js:129:20)
     at _stream_readable.js:908:16
     at process._tickDomainCallback (node.js:381:11)
```

A minor bug because it's handled correctly in the presenter, but it would be better not to dump the stack here.

**Comment by kenperkins** [comment]
Hmm, looking at https://github.com/deconst/content-service/blob/master/src/content.js#L28-34 we're not using `success`, is there a reason for this?

**Comment by kenperkins** [comment]
Wait, that's upload, I just realized this was the download case. Looking more.

**Comment by smashwilson** [comment]
Ah, if I'm listening for the wrong event, that would do it.

**Comment by smashwilson** [comment]
This is actually a prereq for deconst/deconst-docs#87. Otherwise, I'm getting a 500 from the content-service and I can't tell the difference between that and a missing content ID.

**Comment by smashwilson** [comment]
Fixed in deconst/content-service#40. I had to listen for `"complete"` instead of `"end"` and check the statusCode on the response object given to the callback.

No more stack traces, now:

```
2015-06-02T13:02:51.793Z - info: Required resource not found: [404] Error: No layout found for presented URL [http://developer.rackspace.com/blog/categories/openstack/atom.xml]
2015-06-02T13:02:51.798Z - warn: No error layout found for status code [404].
```
